testsuite/features/secondary/trad_cve_audit.feature

```diff
@@ -73,21 +73,21 @@ Feature: CVE Audit on traditional clients
     Then I should see "sle_client" as link
     And I follow "Clear"
 
-  Scenario: List systems by patch status via XML-RPC before patch
+  Scenario: List systems by patch status via API before patch
     When I follow the left menu "Admin > Task Schedules"
     And I follow "cve-server-channels-default"
     And I follow "cve-server-channels-bunch"
     And I click on "Single Run Schedule"
     Then I should see a "bunch was scheduled" text
     And I wait until the table contains "FINISHED" or "SKIPPED" followed by "FINISHED" in its first rows
-    And I am logged in via XML-RPC cve audit as user "admin" and password "admin"
-    When I call audit.list_systems_by_patch_status with CVE identifier "CVE-1999-9979"
+    And I am logged in API as user "admin" and password "admin"
+    When I call audit.list_systems_by_patch_status() with CVE identifier "CVE-1999-9979"
```

Comment:
I'm going to comment it only here, but I think it's on multiple places.
I think using parentheses in our gherkin steps could be dangerous and are very technical, not at all natural language.
I would instead have something like: `I call "audit.list_systems_by_patch_status" method`

Reply:
I would go even further. I am against giving technical details like this in cucumber. This should read:
```
When I list systems with CVE identifier "CVE-1999-9979"
```

I removed a few of these `And I call`, but there are some left. I did not have the courage to remove them all.

As for the parentheses, there were some cases with parentheses and some cases without. I unified with parentheses.

If you insist for me to "fix" this, I would like to change it to full natural language.

Reply:
Well, not insist as part of this PR, but it would be indeed very good contribution.

Reply:
Agree. For later.